src/vm/arm/cgencpu.h

```diff
@@ -103,9 +103,9 @@ EXTERN_C void setFPReturn(int fpSize, INT64 retVal);
 // is guarded by an assert.
 //=======================================================================
 #ifdef _WIN64
-#define MAXFIELDMARSHALERSIZE               40
+#define MAXFIELDMARSHALERSIZE               48
```

Comment:
Is there any possible way to compute these values?

Reply:
Maybe with `constexpr`? The main issue is that we'd want to use all of the field marshalers and compute the max size, which would get really long. I'd be totally fine doing that though if we want to.

Reply:
The fact that static asserts exist makes things okay for me.